lesson_05/jamescapparell/READ.md

```diff
@@ -0,0 +1,9 @@
+# User Stories
+
+## User Story #1
+As a YouTube content creater, I want to be able to view the dislike counter for my video when I am in my YouTube app.
```

Comment:
The user story format should be `As a [ROLE], I want to [ACTION], so that [PURPOSE]`. Please refine these for full credit.